.travis.yml

```diff
@@ -19,7 +20,7 @@ matrix:
   fast_finish: true
   allow_failures:
     # To allow failures, you need to specify the full environment
-    - env: REQUIREMENTS=devel
+    - env: REQUIREMENTS=devel ES_URL=$ES7_DOWNLOAD_URL EXTRAS=all,postgresql,elasticsearch7
```

Comment:
question: It is not enough with `REQUIREMENTS=devel`  it requires the full matrix value to "identify" it?

Reply:
Yes, I noticed that none of the `env` showed in the "Allowed to fail" section and then I noticed the comment above saying that the full environment had to be passed. Once the full environment was passed, that one was allowed to fail :smile: